doc/pmon/pmon-chassis-design.md

```diff
@@ -0,0 +1,658 @@
+# SONiC Chassis Platform Management & Monitoring #
+
+### Rev 1.0 ###
+
+# Table of Contents
+
+  * [Revision](#revision)
+  * [About this Manual](#about-this-manual)
+  * [Scope](#scope)
+  * [Acronyms](#acronyms)
+  * [1. Modular VOQ Chassis Reference](#1-modular-voq-chassis-reference)
+  * [2. SONiC Platform Management and Monitoring](#2-sonic-platform-management-and-monitoring)
+    * [2.1 Functional Requirements](#21-functional-requirements)
+    * [2.2 Chassis Platform Stack](#22-chassis-platform-stack)
+  * [3. Detailed Workflow](#3-detailed-workflow)
+    * [3.1 Chassis Boot Process](#31-chassis-boot-process)
+      * [3.1.1 Supervisor Boot Process](#311-supervisor-boot-process)
+      * [3.1.2 Linecard Boot Process](#312-linecard-boot-process)
+    * [3.2 Chassis Platform Management](#32-chassis-platform-management)
+      * [3.2.1 Midplane Ethernet](#321-midplane-ethernet)
+      * [3.2.2 Chassis Monitoring and ChassisD](#322-chassis-monitoring-and-chassisd)
+      * [3.2.3 Chassis Local Sonic Image Hosting Service](#323-chassis-local-sonic-image-hosting-service)
+      * [3.2.4 Disaggregated vs Global DB](#324-disaggregated-vs-global-db)
+    * [3.3 Peripheral Management](#33-peripheral-management)
+      * [3.3.1 PSUd](#331-psud)
+      * [3.3.2 Thermalctld](#332-thermalctld)
+      * [3.3.3 Xcvrd/SFP](#333-xcvrdsfp)
+      * [3.3.4 LEDd](#334-ledd)
+      * [3.3.5 Syseepromd](#335-syseepromd)
+      * [3.3.6 Midplane Ethernet](#336-midplane-ethernet)
+  * [4. Review Comments](#4-review-comments)
+    * [4.1 Future Items](#41-future-items)
+      
+### Revision ###
+
+ | Rev |     Date    |       Author                                                            | Change Description                |
+ |:---:|:-----------:|:-----------------------------------------------------------------------:|-----------------------------------|
+ | 1.0 |             |  Manjunath Prabhu, Sureshkannan Duraisamy, Marty Lok, Marc Snider       | Initial version                   |
+
+# About this Manual
+This document provides design requirements and interactions between platform drivers and PMON for SONiC on VOQ chassis with line-card CPUs.
+
+# Scope
+For first phase of design, this document covers high level design of the platform support and its interactions with PMON in a VOQ chassis environment. Operations like firmware upgrade will be added at later stage of the development. This document assumes all line-cards and supervisor-cards have a CPU complex where SONiC would be running. It only considers fabric-cards without CPU or SONiC isn't running on Fabric CPU even if it's available. Also, this document assumes line-cards cant be powered on without the Supervisor being operationally up and running. 
```

Comment:
This paragraph is unreadable for me starting from the line "It only considers fabric-cards .... if it's available. " Dont understand what is meant to be said..? Could this part be rephrased? Thanks!

Reply:
Done. thanks.